source/common/stream_info/filter_state_impl.cc

```diff
@@ -1,10 +1,24 @@
+#include "filter_state_impl.h"
 #include "source/common/stream_info/filter_state_impl.h"
 
 #include "envoy/common/exception.h"
 
 namespace Envoy {
 namespace StreamInfo {
 
+std::string FilterStateImpl::serializeAsString() const {
```

Comment:
could you add this to streaminfo dump method? this is incredibly useful for crash debugging!